http/src/main/scala/org/http4s/blaze/http/client/package.scala

```diff
@@ -1,54 +1,3 @@
 package org.http4s.blaze.http
 
-import java.nio.ByteBuffer
-import java.nio.charset.Charset
-
-import org.http4s.blaze.util.{BufferTools, Execution}
-
-import scala.collection.mutable.ArrayBuffer
-import scala.concurrent.Future
-
-
-package object client {
-
-  /** HTTP response received by the client
-    *
-    * @param code Response code
-    * @param status Response message. This have no meaning for the HTTP connection, its just for human enjoyment.
-    * @param headers Response headers
-    * @param body Function for obtaining the response body. Each invocation will return the __next__ body chunk
-    *             with termination signaled by an __empty__ `ByteBuffer` as defined by `ByteBuffer.hasRemaining()`.
-    */
-  case class ClientResponse(code: Int, status: String, headers: Headers, body: () => Future[ByteBuffer]) {
-    def stringBody(): Future[String] = {
-      val acc = new ArrayBuffer[ByteBuffer](8)
-
-      def go(): Future[String] = body().flatMap { buffer =>
-        if (buffer.hasRemaining) {
-          acc += buffer
-          go()
-        }
-        else {
-          val b = BufferTools.joinBuffers(acc)
-          val encoding = getCharset(headers)
-
-          try {
-            val bodyString = Charset.forName(encoding).decode(b).toString()
-            Future.successful(bodyString)
-          }
-          catch { case e: Throwable => Future.failed(e) }
-        }
-      }(Execution.trampoline)
-
-      go()
-    }
-
-    private def getCharset(hs: Headers): String = {
-      headers.collectFirst {
-        case (k, v) if k.equalsIgnoreCase("content-type") => charsetRegex.findFirstIn(v)
-      }.flatten.getOrElse("UTF-8")
-    }
-  }
-
-  private val charsetRegex = "(?<=charset=)[^;]*".r
-}
+package object client {}
```

Comment:
I'm not proud that `ClientResponse` was in a package object. Do we even need the package object, or could we just delete this file?